index.mjs

```diff
@@ -101,9 +103,21 @@ const inputs = getModelInputs(model)
 console.log('Adding model data and inputs to index.js...')
 const indexFile = path.join(targetDir, 'index.js')
 const indexFileContents = fs.readFileSync(indexFile, 'utf8')
+
+let run = `const output = await replicate.run(model, { input });\nconsole.log(output)`;
```

Comment:
This makes me sad. I mean, I know it's our current reality but I feel like users should be able to run official language models with the same code they use to run other models.

@replicate/product how can we make this better?

Reply:
I mean I can simplify this and just fix the model name. But I felt that also using the streaming output made for flashier example.

Reply:
Yeah if you're cool with it can we just fix the language model name here in this PR and have a separate issue/discussion about the different ways of running models?